package.json

```diff
@@ -26,11 +26,11 @@
   "main": "./dist/index.js",
   "dependencies": {},
   "devDependencies": {
-    "@angular/common": "^7.2.3",
-    "@angular/compiler": "^7.2.3",
-    "@angular/compiler-cli": "^7.2.3",
-    "@angular/core": "^7.2.3",
-    "@angular/platform-server": "^7.2.3",
+    "@angular/common": "^9.0.0",
```

Comment:
would be great to make these peerDependencies since I don't think anything in this PR inherently needs 9.x

Reply:
If you can update this in the next 24 hours that would be great. I'll merge and publish tomorrow either way.

Reply:
> would be great to make these peerDependencies since I don't think anything in this PR inherently needs 9.x

Renderer2 requires it, doesn't it?

Reply:
renderer has been deprecated since... I want to say Angular 4. I guess they finally removed it in 9. You can set the peerDependency to be 7.x->9.x (https://npm.github.io/using-pkgs-docs/package-json/types/peerdependencies.html). That way someone who hasn't upgraded to 9 doesn't include all of 9 in their build by accident.

Reply:
I see, but since the PR now includes a reference to `Renderer2` - which is only available in Angular 9.+ - it would throw a missing ref Error when used with Angular v<9. Or am I missing something here?

Reply:
Sorry, been a bit of distracted lately. Renderer2 has been available since Angular 4